dvc/dirs.py

```diff
@@ -20,13 +20,16 @@ def global_config_dir():
     )
 
 
-def site_cache_dir():
+def site_cache_dir(config_site_cache_dir: str | None = None):
```

Comment:
```suggestion
def site_cache_dir(config_site_cache_dir: Optional[str] = None):
```

We still support Python 3.9.